test/verify/machineslib.py

```diff
@@ -381,26 +381,26 @@ def testBasic(self):
         # triangle by status
         b.wait_present("tr.listing-ct-item.listing-ct-nonavigate span span.pficon-warning-triangle-o.machines-status-alert")
         # inline notification with error
-        b.wait_present("tr.listing-ct-panel div.listing-ct-body div.alert.alert-warning")
-        b.wait_in_text("#vm-subVmTest2-last-message", "VM START action failed")
+        b.wait_present("div.alert.alert-danger strong")
+        b.wait_in_text("div.alert.alert-danger strong", "VM subVmTest2 failed to start")
 
         b.wait_present("a.alert-link.machines-more-button") # more/less button
         b.wait_in_text("a.alert-link.machines-more-button", "show more")
         b.click("a.alert-link.machines-more-button")
-        b.wait_present("tr.listing-ct-panel div.listing-ct-body div.alert.alert-warning div > p")
-        b.wait_in_text("a.alert-link.machines-more-button", "show less")
+        b.wait_present("a.alert-link + p")
 
         # the message when trying to start active VM differs between virsh and libvirt-dbus provider
         if (self.provider == "libvirt-dbus"):
-            b.wait_in_text("tr.listing-ct-panel div.listing-ct-body div.alert.alert-warning div > p",
+            b.wait_in_text("a.alert-link + p",
                            "domain is already running")
         else:
-            b.wait_in_text("tr.listing-ct-panel div.listing-ct-body div.alert.alert-warning div > p",
+            b.wait_in_text("a.alert-link + p",
                            "Domain is already active")
 
-        b.click("tr.listing-ct-panel div.listing-ct-body div.alert.alert-warning button") # close button
+        b.wait_in_text("a.alert-link.machines-more-button", "show less")
+        b.click("div.alert.alert-danger button") # close button
         # inline notification is gone
-        b.wait_not_present("tr.listing-ct-panel div.listing-ct-body div.alert.alert-warning")
+        b.wait_not_present("div.alert.alert-danger")
```

Comment:
As apparently our current tests don't pick up the "duplicate indexes" bug, can the test have an "add1 add2 remove1 add3 remove3" scenario and make sure that the right notifications get removed and stay around?

Reply:
Added the test in testBasic.